src/components/postdetail/PostHeader.module.scss

```
@@ -0,0 +1,107 @@
+.container {
+    position: relative; 
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+  }
+  
+  .navigation {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    margin-bottom: 15px;
+    margin-top: 45px;
+  }
+  
+  .text {
+    color: var(--Grey-Grey-500, #9E9E9E);
```

이렇게만 사용하는 것은 작동하지 않나용? (정말 몰라서,,)

```suggestion
    color: --Grey-Grey-500;
```